src/server/src/velvet_dawn/dao/clean.py

```diff
@@ -0,0 +1,9 @@
+import velvet_dawn
+from velvet_dawn.dao import app,db
+
+# When called directly, drops all tables from the DB specified in velvet_dawn.dao
+if __name__ == "__main__":
+    with app.app_context():
+        db.init_app(app)
+        db.drop_all()
+        print("Game state cleaned")
```

Comment:
nit: In future, please use `logger.info`. I'm slowly removing all print statements in favor of logging to make warning/errors logs clearer.

Reply:
Will do, thanks.